packages/fdc3-agent-proxy/test/step-definitions/channels.steps.ts

```diff
@@ -180,14 +178,17 @@ Given('{string} pipes context to {string}', function (this: CustomWorld, context
 
 When('messaging receives {string}', function (this: CustomWorld, field: string) {
   const message = handleResolve(field, this);
-  this.log(`Sending: ${JSON.stringify(message)}`);
-  this.messaging!!.receive(message, this.log);
+  //TODO: restore this
+  //this.log(`Sending: ${JSON.stringify(message)}`);
+  console.log(`Sending: `, message);
+
+  this.messaging!.receive(message, console.log); //this.log);
```

Comment:
Flagging this section, since it looks like it's still a WIP

Reply:
Cleaned up, looks to be in use in quite a few tests.